documentation/docs/_partials/examples/01_get_info/outputs/camelCase.md

```diff
@@ -0,0 +1,33 @@
+```json
```

Comment:
Can't comment at the correct file, but why is there an empty `documentation/docs/_partials/examples/01_get_info/_client_builder.md`?